### PR TITLE
Add more details to the example

### PR DESCRIPTION
**Pull request by adamchalmers**
Responding to @nmldiegues 's comments!

- Add comments
- Add an example of a second ingress rule
- Set initialDelay of 10 seconds for the livenessProbe (for Quicksilver propagation)

### CODE REVIEW

File: named-tunnel-k8s/cloudflared.yaml [hunk 1]
```diff
@@ -75,6 +76,8 @@ data:
     credentials-file: /etc/cloudflared/creds/credentials.json
     # Serves the metrics server under /metrics and the readiness server under /ready
     metrics: 0.0.0.0:2000
+    # Autoupdates applied in a k8s pod will be lost when the pod is removed or restarted, so
+    # autoupdate doesn't make sense in Kubernetes. You should really set this outside Kubernetes though.
```
Comment (nmldiegues) [inline named-tunnel-k8s/cloudflared.yaml]: > You should really set this outside Kubernetes though.

I'd make it slightly less informal:

"We recommend `no-autoupdate: false` when running outside Kubernetes though".

File: named-tunnel-k8s/cloudflared.yaml [hunk 2]
```diff
@@ -84,6 +87,12 @@ data:
     # from the internet to cloudflared, run `cloudflared route dns <tunnel> <hostname>`.
     # E.g. `cloudflared route dns example-tunnel tunnel.example.com`.
     ingress:
+    # The first rule proxies traffic to the httpbin sample Service defined in app.yaml
     - hostname: tunnel.example.com
       service: http://web-service:80
+    # This rule sends traffic to the built-in hello-world HTTP server. This lets you check if cloudflared
+    # is up, without needing to use kubectl (e.g. if you're on mobile)
```
Comment (nmldiegues) [inline named-tunnel-k8s/cloudflared.yaml]: Nothing against "pimping" this with the hello world service.  But why wouldn't the user be able to reach `tunnel.example.com` on his mobile as well?

File: named-tunnel-k8s/cloudflared.yaml [hunk 3]
```diff
@@ -31,6 +31,7 @@ spec:
             path: /ready
             port: 2000
           failureThreshold: 1
+          initialDelaySeconds: 10
```
Comment (nmldiegues) [inline named-tunnel-k8s/cloudflared.yaml:34]: This is a start, but I'm not sure 10sec is enough. Under normal circumstances the edge propagation seems to be up to 20sec, so I'd be happier with 30 here.

File: named-tunnel-k8s/cloudflared.yaml (hunk 2 above)
Comment (adamchalmers) [reply named-tunnel-k8s/cloudflared.yaml]: If cloudflared -> origin service is disrupted. I'll explain in the comment.

**Review by nmldiegues — APPROVED** [review]

**Pull request merged**